bin/pycbc_optimize_snr

```diff
@@ -137,6 +139,21 @@ parser.add_argument('--output-path', required=True,
                     help='Path to a directory to store results in')
 parser.add_argument('--cores', type=int,
                     help='Restrict calculation to given number of CPU cores')
+parser.add_argument('--optimizer', type=str, default='differential_evolution',
+                    help='The optimizer to use, shgo or differential_evolution')
+parser.add_argument('--maxiter', type=int, default='50',
+                    help='Differential Evolution: The maximum number of '
+                         'generations over which the entire population is evolved.')
+parser.add_argument('--popsize', type=int, default='100',
+                    help='Differential Evolution: A multiplier for setting the '
+                         'total population size.')
+parser.add_argument('--iters', type=int, default='3',
+                    help='shgo: Number of sampling points used in the construction '
+                         'of the simplicial complex. ')
+parser.add_argument('--n', type=int, default='76',
```

Comment:
I suggest looking for more descriptive names for these options, for example:
* Prefix the DI-relevant options with "di", and the SHGO-relevant options with "shgo". 
* Is there a better (but still short) name than "n"? Does it make sense to recycle the --maxiter option for this parameter?
* Please clarify in the help strings that some options are only relevant to DI and SHGO (e.g. "Only relevant with --optimizer differential_evolution").

Reply:
* When the argument is an int, you do not need quotes for the `default` kwarg, since it is an int.

Reply:
With regards to the second point, I had mixed up the two parameters 🤦‍♂️. I can't think of a shorter term for n, with it's definition being 'Number of sampling points used in the construction of the simplicial complex.'. I think maybe I will go with 'shgo_samples'.

Reply:
Then `--shgo-num-points` seems a reasonable choice to me, as does `--shgo-samples` (but please stick to dashes).

It seems I was the one mixing up stuff! I meant to propose reusing `--maxiter` in place of `--iters`, not `--n`.